### PR TITLE
Upgrade to Bourbon 5

### PR DESCRIPTION
**Pull request by tysongach**
This PR introduces compatibility with Bourbon 5 (currently in beta and nearing release). Bourbon 5 introduces quite a few breaking changes and so upgrading would mean Bourbon 4 is no longer supported.

### CODE REVIEW

File: source/stylesheets/refills/_vertical-tabs.scss [hunk 1]
```diff
@@ -14,7 +14,7 @@
   $tab-content-background: lighten($tab-border-color, 10%);
   $tab-active-color: $tab-content-background;
   $tab-inactive-color: $base-background-color;
-  $vertical-tabs-height: em(300);
+  $vertical-tabs-height: 18.75em;
```
Comment (kaishin) [inline source/stylesheets/refills/_vertical-tabs.scss]: Not so sure about this, but I have no better idea.

**Comment by kaishin** [comment]
One minor comment otherwise looking good.

File: source/stylesheets/refills/_vertical-tabs.scss (hunk 1 above)
Comment (tysongach) [inline source/stylesheets/refills/_vertical-tabs.scss]: Totally agree.

For now, I didn’t want to introduce any visual changes so I did a direct conversion. Eventually, I believe Refills should strive to not declare fixed sizes and leverage a more flexible system by using flexbox and other means.

**Comment by tysongach** [comment]
Thanks, @kaishin!

**Comment by ACPK** [comment]
@kaishin Think it will get merged?

**Pull request merged**